tdf-guidance.md

```diff
@@ -107,12 +107,12 @@ The following table shows the target device families to declare in the manifest:
   <tr>
     <td class="tg-yw4l" rowspan="2">Windows</td>
     <td class="tg-yw4l">7</td>
-    <td class="tg-baqh" colspan="2">Windows7.Desktop</td>
+    <td class="tg-baqh" colspan="2">MsixCore.Desktop</td>

```

Comment:
><td class="tg-baqh" colspan="2">MsixCore.Desktop</td> [](start = 4, length = 53)

I think you can avoid having two td tags with MsixCore.Desktop by adding rowspan="2" here and deleting the other one